### ISSUE TITLE
Sometimes Docker did not successfully start

### ISSUE THREAD

**Issue by ailispaw** [bargees/barge-os#6]
https://github.com/aberresch/udre/blob/master/Vagrantfile#L91

**Comment by ailispaw** [comment]
Hi @aberresch,

I found a bug and fixed it at #7.
I will include this in the next release.
Thank you for letting me realize and using Barge anyway.

**Comment by ailispaw** [comment]
For a while you can replace the script as below in your Vagrantfile.

```bash
      wget -qO /tmp/docker https://raw.githubusercontent.com/bargees/barge-os/master/overlay/etc/init.d/docker
      chmod +x /tmp/docker
      mv /tmp/docker /etc/init.d/docker
```
